src/style.css

```diff
@@ -2,0 +1,114 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url(./fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
Include a fallback font-family in case the main one doesn't work, for example: 'font-family: Avenir, Arial, sans-serif;'. This is a good practice for ensuring text remains readable if the custom font fails to load.